File: cs/proxy/db/HallPlayerInfoOperation.cs
/// <summary>
/// 玩家信息操作
/// </summary>
namespace org.zgl.service.client.hall{
	[Proxy(gameId = -7)]
	public interface HallPlayerInfoOperation : IHttpSyncService { 
		/// <summary>
		/// 查看礼物
		/// </summary>
		GiftDto playerGiftInfo(long arg0);
		/// <summary>
		/// 查看个人信息
		/// </summary>
		PlayerInfoDto playerInfo(long arg0);
		/// <summary>
		/// 查看道具
		/// </summary>
		PropDto playerPropInfo(long arg0);
	}
}

File: componet/src/main/java/org/zgl/service/client/hall/HallPlayerInfoOperation.java
package org.zgl.service.client.hall;

import org.zgl.dto.clinet.hall.GiftDto;
import org.zgl.dto.clinet.hall.PlayerInfoDto;
import org.zgl.dto.clinet.hall.PropDto;
import org.zgl.proxy.ClassDesc;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.IHttpSyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/21
 * @文件描述：
 */
@ClassDesc("玩家信息操作")
public interface HallPlayerInfoOperation extends IHttpSyncService {
    @MethodDesc("查看个人信息")
    PlayerInfoDto playerInfo(long uid);
    @MethodDesc("查看礼物")
    GiftDto playerGiftInfo(long uid);
    @MethodDesc("查看道具")
    PropDto playerPropInfo(long uid);
}
